UserExperience_Remove_CssJs_Extension.php

```diff
@@ -55,7 +55,7 @@ public function run() {
 
 		Util_Bus::add_ob_callback( 'removecssjs', array( $this, 'ob_callback' ) );
 
-		add_filter( 'w3tc_save_options', array( $this, 'w3tc_save_options' ) );
+		add_action( 'w3tc_config_ui_save-w3tc_userexperience', array( $this, 'w3tc_config_ui_save_w3tc_userexperience' ), 10, 2 );
```

Comment:
The callback function name is long.  We try to keep names limited to 3 parts when possible.
  It also messes up the history of the method in Git.